lib/plop-pack/skel/component/README.md.hbs

```diff
@@ -42,20 +41,26 @@ pnpm install
 
 ### Testing
 
+Run the `jest` unit tests.
+
 ```shell
 npm test
 ```
 
 
 ### Building
 
+Compile the TypeScript into plain old JavaScript.
```

Comment:
How about?
> Build the package by compiling the TypeScript source code.

I'd like to avoid 'plain old' because JavaScript has many forms.

Also we compile to a JavaScript that our webpack can understand and probably ES2020 rather than 'plain old'.

Reply:
Updated!